Wappscript.js

```diff
@@ -0,0 +1,71 @@
+const windSpeedMap = new Map();
+const temperatureMap = new Map();
+
+
+fetch("http://localhost:8000/windspeed.json")
```

Comment:
Usually it's a better practice to keeping all the constants like this in another place like configs.js. It makes refactoring/maintainability way better. Then it's very clear for future developers where they should edit in case the server URL changes. 

(Another reading: For numbers it's also commonly said that "do not use magic numbers": http://wiki.c2.com/?MagicNumber)